### ISSUE TITLE
bootstrap-theme.css

### ISSUE THREAD

**Issue by richardwo** [mizzao/meteor-bootstrap-3#1]
bootstrap-theme.css is included is included in the css folder, but the CSS that Meteor generates doesn't include the theme.

**Comment by mizzao** [comment]
Yeah, that file is not added in https://github.com/mizzao/meteor-bootstrap-3/blob/master/package.js.

What is it supposed to do? I think you are free to add your own theme in your app by copying that file or generating a similar one, but I'm not sure if we want to force it on people in this package. It's a much smaller set of CSS than the main bootstrap CSS, and can be used to configure specific styling.

**Comment by richardwo** [comment]
The theme enhances the base css to include gradients on buttons and
other components.
Sorry - got confused because the file was in your repo - you are right
not to force it on people.
I'll just copy the bootstrap-theme.css file into my site folder and
Meteor should do the rest.
Thanks.

----- Original Message -----

**Comment by mizzao** [comment]
If this is a standard theme, happy to include it - otherwise, if it's something that each user downloads a custom version of, I'll leave it out. Could you point me to some docs on this?

**Comment by simison** [comment]
Theme is optional and it shouldn't be included by default in this kind of package. 

One day when Meteor packages will be actually smart, we could have opt-in conditional to include it.

See mentions to it at http://getbootstrap.com/getting-started/
